drafts/taking-a-byte-of-the-carbon-cake.md

```diff
@@ -0,0 +1,291 @@
+something of which our customers are increasingly business-conscious.
+
+### Not in Scope
+
+Our focus is on the carbon cost of systems that run the software that we create, and we highlight just a few of many worthwhile patterns that are applicable. 
+Although we touch lightly on many topics, intentionally out of scope is the way in which cloud providers account for their carbon footprints. 
+Also note that the relative carbon footprints of everyday things are necessarily a guideline only.
+
+## What is ‘carbon cost’?
+First, let’s review the carbon cost of some popular items so that we can understand the relative cost of common software components. For this we’ll use the standard unit of Kg of carbon dioxide (KgCO2). Note that for greenhouse gases other than CO2 it is common practice to convert to CO2 as an equivalent.
+
+![Figure: Relative carbon footprints of everyday thing](https://cdn.hashnode.com/res/hashnode/image/upload/v1730803114790/O_AEqKW9y.png?auto=format)
+
+In this chart we show some examples of everyday carbon footprints; from the lowest footprint of 10 KgCO2 for a train journey London to Birmingham, and 
+similar for a bus at 130 KgCO2, scaling up to the cost of a petrol lawnmower for 22 mows/year, some example flight costs, pets and the energy cost for a house.
+
+Some details behind these numbers: the cost of a house represents an average UK gas-fired heating with grid electricity. 
+The cow is calculated methane production per year and uses standard conversion to CO2. Costs of transport are factored for one person journey, whereas costs showing 'p.a' represent 1 year. 
+Cost of ownership of cats and dogs represent their whole-ownership life-cost per year. Source references are provided at the end of this article.
+
+
+### Aside: How much is 1kg of CO2?
+
+![Figure: A 1m balloon of 1kg CO2](https://cdn.hashnode.com/res/hashnode/image/upload/v1730803843903/zOJqgrkgS.jpg?auto=format)
+
+*Figure: A 1m balloon of 1kg CO2*
+
+As an aside, if you are a visual person, to picture 1Kg of CO2, imagine a balloon of approximately 1 m diameter, at 18 degrees, sea level
+
+## How do we measure?
+
+To calculate an equivalent carbon cost of software solutions deployed to the cloud, we must consider the triad of compute, memory and disk. We will ignore the embedded cost of hardware because cloud providers include that in the rate. Note that if we wish to measure for physical hardware solutions such as on-premise data centres or a Mac or laptop, of course we need to factor in the embedded carbon cost.
+
+There’s plenty of controversy over climate reporting, with some hypervisor providers reputed to adopt clever accounting methods to obfuscate the true carbon cost. There are some key omissions which force us to guess some aspects, such as whether AWS S3 storage is based on physical hard drives, or how long a Lambda function remains in memory.
```

Comment:
I would slightly refomulate this paragraph and focus on the "lack of transparency" of cloud providers using different methodologies. The "controversy over climate reporting" is an ever bigger topic.